app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +25,8 @@
         android:name="android.hardware.location"
         android:required="false" />
 
+    <uses-sdk tools:overrideLibrary="com.pierfrancescosoffritti.androidyoutubeplayer"/>
```

Comment:
this line doesn't fit in this PR now since you are defining it in the tutorial module. But this will end up showing back up with the tool details UI, so it can stay